backend/src/User.ts

```diff
@@ -38,6 +74,21 @@ export class User {
     EventBus.emit(User.EVENTS.signedIn, this.credentials)
   }
 
+  async authHashLogin(username: string, authhash: string): Promise<any> {
+    return instance
+      .post<IRawUser>('/user/login/authhash', { username, authhash })
+      .then((resp:any) => this.process(resp, username))
+  }
```

Comment:
Would it be okay to use async await here?

Reply:
done, please review